apps/omg_watcher_rpc/priv/swagger/info_api_specs.yaml

```diff
@@ -525,6 +527,149 @@ paths:
                         description: Something went wrong on the server
                         messages:
                           error_key: error_reason
+  /block.all:
+    post:
+      tags:
+        - Block
+      summary: Gets all blocks (with a limit on the number of blocks to return).
```

Comment:
I guess implementation enforces some order on the returned items which is not mentioned.
Also you can and I encourage you to make longer `description` of this path (see transaction.all)

Reply:
Added in https://github.com/omisego/elixir-omg/pull/1192/commits/e9d194f37d5b9cb3f69ff3f35cfa7b4d587f0bc8

I see the descriptions are quite different for each endpoint, so I'll try to follow a format:

```yaml
description: >
  Behavior
  Usage
  Remarks
```